src/com/github/mtdrewski/GRAPH_moment/model/graphs/Vertex.java

```diff
@@ -24,6 +24,14 @@ public int id() {
         return id;
     }
 
+    public void setxPos(float x){
```

Comment:
Mayby a single function setPosition(x, y) would be better? (Shorter for sure)
As far as I know, we always set x and y in the same time